File: Easy leetcode/1637. Widest Vertical Area Between Two Points Containing No Points.py
class Solution:
    def maxWidthOfVerticalArea(self, points):
        x_axis = [point[0] for point in points]

        x_axis.sort()

        width = 0
        for i in range(1, len(x_axis)):
            width = max(width, x_axis[i] - x_axis[i - 1])

        return width

File: Easy leetcode/1637. Widest Vertical Area Between Two Points Containing No Points.java
import java.util.*;

class Solution
{
    public int maxWidthOfVerticalArea(int[][] points)
    {
        int[] x_axis = new int[points.length];

        for (int i = 0; i < points.length; i++)
        {
            x_axis[i] = points[i][0];
        }

        Arrays.sort(x_axis);

        int width = 0;
        for (int i = 1; i < x_axis.length; i++)
        {
            width = Math.max(width, x_axis[i] - x_axis[i - 1]);
        }

        return width;
    }
}

